### PR TITLE
Feature/#68: Theme upgrades for Drupal 10

### PR DESCRIPTION
**Pull request by smillart**
This pull request contains theme upgrades regarding detected **Drupal 10** compatibility errors/warnings.

### CODE REVIEW

**Comment by smillart** [comment]
The `core_version_requirement` key must be present in `kiso.info.yml` as well as in the STARTERKIT.

**Comment by smillart** [comment]
**Warning 2:** The `spaceless` tag is deprecated since Twig 2.7. Using the "spaceless" filter with the `apply` tag instead.

**Comment by smillart** [comment]
**Theme checked with the "[Drupal 10 upgrade status](https://www.drupal.org/project/upgrade_status)" module... 5 warnings found:**

1. Call to deprecated function `render()`. Deprecated in "drupal:9.3.0" and is removed from "drupal:10.0.0". Use of `Drupal\Core\Render\RendererInterface::render()` instead
2. Call to deprecated function `drupal_get_path()`. Deprecated in "drupal:9.3.0" and is removed from "drupal:10.0.0". Use of `Drupal\Core\Extension\ExtensionPathResolver::getPath()` instead.
3. The `core_version_requirement` key must be present in `kiso.info.yml` as well as in the STARTERKIT,
4. The `spaceless` tag is deprecated since Twig 2.7. Using the "spaceless" filter with the `apply` tag instead,
5. The `drupal/core` requirement is not compatible with the next major version of Drupal. Update it to be compatible.

File: kiso.info.yml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 # Defines theme information
 name: 'Kiso 基礎'
-description: 'Base theme for Drupal 8 websites build around accessibility and simplicity.'
+description: 'Base theme for Drupal 8/9/10 websites build around accessibility and simplicity.'
 type: theme
 base theme: false
-version: '8.x-2.x-dev'
-core: 8.x
-core_version_requirement: ^8 || ^9
+core_version_requirement: ^8 || ^9 || ^10
```
Comment (rutiolma) [inline kiso.info.yml:6]: can we remove D8 reference? We don't need this legacy compatibility

**Pull request merged**